oscap-anaconda-addon.spec

```diff
@@ -48,9 +63,12 @@ make install DESTDIR=%{buildroot}
 %files -f %{name}.lang
 %{_datadir}/anaconda/addons/org_fedora_oscap
 
-%doc COPYING ChangeLog README
+%doc COPYING ChangeLog README.md
```

Comment:
Update `Makefile` to contain `README.md` instead of `README`. This should fix the issue jenkins reports.

Reply:
@dahaic done